### PR TITLE
Golf bytes

### PR DESCRIPTION
**Pull request by sventschui**
Golf some bytes

-26B

### CODE REVIEW

File: src/diff/index.js [hunk 1]
```diff
@@ -352,7 +352,7 @@ export function unmount(vnode, ancestorComponent, skipRemove) {
 		}
 	}
 
-	if (dom!=null) removeNode(dom);
+	if (dom) removeNode(dom);
```
Comment (JoviDeCroock) [inline src/diff/index.js]: We should test this with an empty textNode/textNode with 0. Not entirely sure if this would still work

Comment (sventschui) [reply src/diff/index.js]: I'll try to dig into this later

File: src/diff/index.js [hunk 2]
```diff
@@ -328,12 +328,12 @@ export function unmount(vnode, ancestorComponent, skipRemove) {
...
 	let dom;
 	if (!skipRemove && vnode._lastDomChild==null) {
-		skipRemove = (dom = vnode._dom)!=null;
+		skipRemove = (dom = vnode._dom);
 	}
 
 	vnode._dom = vnode._lastDomChild = null;
 
-	if ((r = vnode._component)!=null) {
+	if ((r = vnode._component)) {
```
Comment (sventschui) [inline src/diff/index.js]:
```suggestion
	if (r = vnode._component) {
```

File: src/diff/index.js (hunk 1 above)
Comment (developit) [reply src/diff/index.js]: if `dom instanceof Text` it should work fine. IIRC the reason for using `!=null` and `==null` all over the codebase was performance + byte size. We'll want to check for perf regressions if we swap them, and make sure we use them uniformly.

**Comment by developit** [comment]
@andrewiggins exactly. The "no change" and "repeat diff" benchmarks can sometimes show these, but it'd be better to run them through the main public benchmarks.

**Comment by sventschui** [comment]
Closing this as per the perf remarks 👍

**Pull request closed without merging**